tools/sgen/input/FluentUIStyle.yml

```diff
@@ -439,6 +440,12 @@ AP_MSFListTokens:
 
 MSFIconOnlyListTokens extends MSFListTokens:
   iconColor: $Colors.Foreground.neutral3
+  
+AP_MSFListHeaderFooterTokens:
+  backgroundColor: [ rest: $Colors.Background.neutral1, highlighted: $Colors.Background.neutral5 ]
+  textColor: [ default: $Colors.Foreground.neutral3, bolded: $Colors.Foreground.neutral1 ]
```

Comment:
nit: regarding the "bolded" naming. Can you double check that with the design team? The reason I bring this up is that it can be misleading if it's implying a bold font and later needs to be changed to something that is not bold.